### PR TITLE
Removes All CamelCase'd "NanoTrasen"

### PR DESCRIPTION
**Pull request by matttheficus**
## What Does This PR Do
Standardizes all mentions of Nanotrasen to not be CamelCase'd as per Lore Team.

Adds a reminder to the Other Notes section of the Contributor's Guide.

Adds CI check for CamelCase on Nanotrasen (thanks @Contrabang)

## Why It's Good For The Game
Standardization good.

_Why the FUCK is there a VAR EDITED mention of this on MetaStation god damn that map makes me WANT to hate it._

## Changelog
:cl: matttheficus, contrabang
spellcheck: ensures proper standardizations for all mentions of "Nanotrasen".
/:cl:

### CODE REVIEW

**Review by Burzah — APPROVED** [review]

**Review by Contrabang — APPROVED** [review]
LGTM

File: tools/ci/check_grep2.py [hunk 1]
```diff
@@ -89,6 +89,11 @@ def check_proc_args_with_var_prefix(lines):
         if PROC_ARGS_WITH_VAR_PREFIX_RE.match(line):
             return Failure(idx + 1, "Changed files contains a proc argument starting with 'var'.")
 
+NANOTRASEN_CAMEL_CASE = re.compile(r"NanoTrasen")
+def check_for_nanotrasen_camel_case(lines):
+    for idx, line in enumerate(lines):
+        if NANOTRASEN_CAMEL_CASE.search(line):
+            return Failure(idx + 1, "Nanotrasen should not be spelled in the camel case form.")
 TO_CHAT_WITH_NO_USER_ARG_RE = re.compile(r"to_chat\(\"")
```
Comment (Contrabang) [inline tools/ci/check_grep2.py:97]:
```suggestion
        if NANOTRASEN_CAMEL_CASE.search(line):
            return Failure(idx + 1, "Nanotrasen should not be spelled in the camel case form.")
            
TO_CHAT_WITH_NO_USER_ARG_RE = re.compile(r"to_chat\(\"")
```

**Review by Contrabang — COMMENTED** [review]
One last change, glad I made the CI lol (manuals.dm 152)
image

**Review by Contrabang — APPROVED** [review]
LGTM

**Review by dearmochi — APPROVED** [review]
Not sure how I feel about going as far as adding a check in CI, but LGTM

**Review by S34NW — CHANGES_REQUESTED** [review]
I don't think we need to spend CI time on this sort of thing, rest looks fine though

**Comment by AffectedArc07** [comment]
> I don't think we need to spend CI time on this sort of thing, rest looks fine though

The CI time is minimal, especially when considering this proble will happen again without it.

Keep the CI.

**Review by S34NW — APPROVED** [review]

**Pull request merged**